src/libs/ajax/GoogleStorage.ts

```diff
@@ -76,7 +76,7 @@ const checkRequesterPaysError = async (response): Promise<RequesterPaysErrorInfo
 };
 
 export const responseContainsRequesterPaysError = (responseText) => {
-  return _.includes('requester pays', responseText);
+  return _.includes('to user project', responseText);
```

Comment:
This function needs to know how to recognize requester-pays problems from the Rawls `bucketOptions` API. I don't know the best way to do this; "to user project" is probably way too generic